src/OrbitQt/main.cpp

```diff
@@ -62,6 +62,9 @@ ABSL_DECLARE_FLAG(std::string, collector);
 ABSL_DECLARE_FLAG(bool, local);
 ABSL_DECLARE_FLAG(bool, devmode);
 ABSL_DECLARE_FLAG(bool, nodeploy);
+ABSL_FLAG(std::string, path_to_capture_file, "",
+          "Path to the capture file. If empty, skip ProfilingTargetDialog and start "
+          "orbitmainwindow with the given capture file.");
```

Comment:
Sorry, I just realize that this should probably be done in a different way. Like this, opening a capture from file only works when the filename is explicitly specified as "-path_to_capture_file", right?

If we enable loading a capture with command line args, we should do it the proper way so it also works when one or multiple .orbit files are dropped onto the .exe in windows, or if .orbit is associated with Orbit and a file is opened by double-click. Windows handles this by simply adding all filenames as unnamed parameters, not sure about Linux.

I don't know if absl can handle this, or if we need to parse the arguments ourselves, but I suggest you adjust the code and filtering of parameters to interpret all unnamed parameters as filenames, spawning multiple Orbit instances if more than one filename is given.

Reply:
Thanks for the suggestion. I removed this flag and also added the `void FilterCommandLineArgs(const std::vector<std::string>& command_line_args, QStringList& command_line_flags, std::vector<std::string>& capture_file_paths)` the method in main.cpp to extract capture file paths and the command line flags.